src/param.c

```diff
@@ -33,6 +33,10 @@ static void init_param_float(param_id_t id, char name[PARAMS_NAME_LENGTH], float
 // function definitions
 void init_params(void)
 {
+  for(uint8_t i = 0; i < PARAMS_COUNT; i++)
+  {
+    init_param_int(i, "DEFAULT", 0);
+  }
```

Comment:
Yeah, it's because it seems that if the parameters are not initialized, then they don't get sent to fcu_io.  Instead, fcu_io just keeps requesting parameters because it doesn't get them all.

I put that in there mainly as a way to ensure that fcu_io got all the paramters, and so I could see which parameters weren't being initialized.  I think now that I have made sure that param.h and param.c agree on the number of parameters we can get rid of this.